app/models/organization.rb

```diff
@@ -43,6 +45,24 @@ class Organization < VitaPartner
 
   default_scope -> { includes(:child_sites).order(name: :asc) }
   alias_attribute :allows_greeters?, :allows_greeters
+  scope :with_capacity, -> do
+    not_these_states = ['intake_before_consent', 'intake_in_progress', 'intake_greeter_info_requested', 'intake_needs_doc_help', 'file_mailed', 'file_accepted', 'file_not_filing', 'file_hold', 'file_fraud_hold']
```

Comment:
Consider using `TaxReturnStateMachine::EXCLUDED_FROM_CAPACITY`

```
irb(main):001:0> TaxReturnStateMachine::EXCLUDED_FROM_CAPACITY
=> [:intake_before_consent, :intake_in_progress, :intake_greeter_info_requested, :intake_needs_doc_help, :file_mailed, :file_accepted, :file_not_filing, :file_hold, :file_fraud_hold]
```

Reply:
i was hoping something like that existed and i'm glad i could just push it up the dumb way and let market forces sort it out